service/src/test/java/bio/terra/workspace/pact/TpsApiTest.java

```diff
@@ -185,27 +188,25 @@ public void deletingAnExistingPolicy() throws Exception {
   public RequestResponsePact getPaoWithAnExistingPolicy(PactDslWithProvider builder) {
     var policyResponseShape =
         new PactDslJsonBody()
-            .valueFromProviderState("objectId", existingPolicyProviderStateValue, existingPolicyId)
+            .stringValue("objectId", existingPolicyId.toString())
             .stringMatcher("component", tpsComponentRegex)
             .stringMatcher("objectType", tpsObjectTypeRegex)
             .object("effectiveAttributes", tpsPolicyInputsObjectShape)
             .object("attributes", tpsPolicyInputsObjectShape);
 
     return builder
-        .given(existingPolicyState)
+        .given("a policy with the given id exists", Map.of("id", existingPolicyId.toString()))
```

Comment:
This is how you tell the provider that a policy with a specific ID is expected to exist.

On the provider side tests in TPS, you'll end up with code like:

 @State({"a policy with the given id exists"})
  public void policyExists(Map<?, ?> parameters) throws InterruptedException {
    var existingPolicyId = parameters.get("id").toString();
    // TODO: mock/stub out DAO to return a hardcoded policy whenever `existingPolicyId` is requested
  }

Reply:
Didn't end up needing to do this after all, but the comment is still valid if this construct creeps back in down the road.